File: Practice Arrays/7Largest and Second Largest.py
t = int(input())

while t > 0:
    n = int(input())
    a = list(map(int, input().split()))
    t -= 1
    # Your code goes here
    first_max=0
    second_max=0
    for i in range(n):
        if a[i]>first_max:
            second_max=first_max
            first_max=a[i]
        elif a[i]!=first_max and a[i]>second_max:
            second_max=a[i]
    print(first_max+second_max)
            

File: Practice Arrays/codigo_cs/7Largest and Second Largest.cs
using System;

public class Test7
{
	public static void TwoLargest()
	{
		// your code goes here
		int t = int.Parse(Console.ReadLine());
		for (int _ = 0; _ < t; _++)
		{
			int n = int.Parse(Console.ReadLine());
			int[] a = Console.ReadLine().Split().Select(int.Parse).ToArray();
			int max1 = 0, max2 = 0;
			for (int i = 0; i < n; i++)
			{
				if (a[i] > max1)
				{
					max2 = max1;
					max1 = a[i];
				}
				else if (a[i]!=max1 && a[i] > max2)
				{
					max2 = a[i];
				}
			}
			Console.WriteLine($"result {max1+ max2}");
		}
	}
}
